tests/PfsFiberTrace.py

```diff
@@ -0,0 +1,202 @@
+        self.assertEqual(pfsFiberTrace.xHigh, pfsFiberTraceNew.xHigh)
+        self.assertEqual(pfsFiberTrace.nCutLeft, pfsFiberTraceNew.nCutLeft)
+        self.assertEqual(pfsFiberTrace.nCutRight, pfsFiberTraceNew.nCutRight)
+        self.assertEqual(pfsFiberTrace.interpol, pfsFiberTraceNew.interpol)
+        self.assertEqual(pfsFiberTrace.swathLength, pfsFiberTraceNew.swathLength)
+        self.assertEqual(pfsFiberTrace.overSample, pfsFiberTraceNew.overSample)
+        self.assertEqual(pfsFiberTrace.maxIterSF, pfsFiberTraceNew.maxIterSF)
+        self.assertEqual(pfsFiberTrace.maxIterSig, pfsFiberTraceNew.maxIterSig)
+        self.assertEqual(pfsFiberTrace.lambdaSF, pfsFiberTraceNew.lambdaSF)
+        self.assertEqual(pfsFiberTrace.lambdaSP, pfsFiberTraceNew.lambdaSP)
+        self.assertEqual(pfsFiberTrace.lambdaWing, pfsFiberTraceNew.lambdaWing)
+        self.assertEqual(pfsFiberTrace.lSigma, pfsFiberTraceNew.lSigma)
+        self.assertEqual(pfsFiberTrace.uSigma, pfsFiberTraceNew.uSigma)
+
+        """Create new FiberTraceSet"""
+        ftsNew = makeFiberTraceSet(pfsFiberTraceNew, self.flat.getMaskedImage())
+
+        self.assertEqual(fiberTraceSet.size(), ftsNew.size())
+        for iFt in range(ftsNew.size()):
+            ft = fiberTraceSet.getFiberTrace(iFt)
+            ftNew = ftsNew.getFiberTrace(iFt)
+            ftf = ft.getFiberTraceFunction()
+            ftfNew = ftNew.getFiberTraceFunction()
+            ftfc = ftf.fiberTraceFunctionControl
+            ftfcNew = ftfNew.fiberTraceFunctionControl
+            self.assertEqual(ft.getITrace(), ftNew.getITrace())
+            self.assertEqual(ft.isTraceSet(), ftNew.isTraceSet())
+            self.assertEqual(ft.isProfileSet(), ftNew.isProfileSet())
+            self.assertEqual(ft.isFiberTraceProfileFittingControlSet(), ftNew.isFiberTraceProfileFittingControlSet())
+            self.assertEqual(ft.getWidth(), ftNew.getWidth())
+            self.assertEqual(ft.getHeight(), ftNew.getHeight())
+            self.assertEqual(ftf.yCenter, ftfNew.yCenter)
+            self.assertEqual(ftf.xCenter, ftfNew.xCenter)
+            self.assertEqual(ftf.yLow, ftfNew.yLow)
+            self.assertEqual(ftf.yHigh, ftfNew.yHigh)
+
+            self.assertEqual(ftfc.interpolation, ftfcNew.interpolation)
+            self.assertEqual(ftfc.order, ftfcNew.order)
+            self.assertEqual(ftfc.xLow, ftfcNew.xLow)
+            self.assertEqual(ftfc.xHigh, ftfcNew.xHigh)
+            self.assertEqual(ftfc.nPixCutLeft, ftfcNew.nPixCutLeft)
+            self.assertEqual(ftfc.nPixCutRight, ftfcNew.nPixCutRight)
+            self.assertEqual(ftfc.nRows, ftfcNew.nRows)
+
+            coeffs = ft.getTraceCoefficients()
+            coeffsNew = ftNew.getTraceCoefficients()
+            self.assertEqual(coeffs.shape[0], coeffsNew.shape[0])
+            for iCoeff in range(coeffs.shape[0]):
+                self.assertAlmostEqual(coeffs[iCoeff], coeffsNew[iCoeff], places=3)
```

Comment:
Why are you using `assertAlmostEqual` here?  Shouldn't the coefficients be copied exactly?